x-pack/plugins/cases/common/utils/attachments.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CommentRequest,
+  CommentRequestExternalReferenceType,
+  CommentType,
+  CommentRequestPersistableStateType,
+} from '../api';
+
+/**
+ * A type narrowing function for external reference attachments.
+ */
+export const isCommentRequestTypeExternalReference = (
```

Comment:
I moved them to the common folder because both the backend and the front end needs them.